akka-docs-dev/rst/scala/http/routing-dsl/directives/form-field-directives/formFields.rst

```diff
@@ -3,8 +3,6 @@
 formFields
```

Comment:
And: In the light of #17938 we should maybe add a sentence explaining that there is nothing in this directive which filters for POST requests or any other request method type. It only concerns itself with providing convenient deserialization access to the request entity.